### PR TITLE
Make review + Latex slides 116 - 150 review

### PR DESCRIPTION
**Pull request by mennthor**
Reviewed Make part and Make example.
- Changed some epxressions
- Fixed some typos
- Added clean target to exercise

Reviewed Latex slides 116 - 150.
- Reviewed typographie, texorstring, links, debug, makros, width and math_expert
- Changed some exporessions
- Fixed some typos

### CODE REVIEW

File: latex/content/debug.tex [hunk 1]
```diff
@@ -1,6 +1,6 @@
 \subsection{Debug}
 
-\begin{frame}[fragile]{\lstinline+\\overfullrule+}
+\begin{frame}[fragile]{\texttt{\textbackslash overfullrule}}
```
Comment (MaxNoe) [inline latex/content/debug.tex]: Why did you change this? `\lstinline` should be correct for code examples!

Comment (mennthor) [inline latex/content/debug.tex]: It's hard to read in the slide title, because its red on grey.
White typewriter is much better for the title

Comment (mennthor) [inline latex/content/debug.tex]: Or we change, that inline code is dispayed white instead of red in the title only.

Comment (MaxNoe) [inline latex/content/debug.tex]: Ah, ok. So it's the font color. Yes, I would prefer to change the appearance of lstinline in titles. Let me check how to do it.

Comment (mennthor) [inline latex/content/debug.tex]: Changed back to lstinline and added white color by hand in every title.

Comment (itoijala) [inline latex/content/debug.tex]: I can remember fixing the font color for some kind of titles, I think it was the box titles.

Ah yes, there it is: https://github.com/pep-dortmund/toolbox-workshop/blob/master/common/settings.tex#L16

You should be able to use beamer styles to do the same for frame titles.

File: exercises-toolbox/5-make/aufgabe.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
-Schreibe eine Makefile, die beim Aufruf von `make` die Dateien plot.pdf und parameter.txt erstellt.
-Sie sollte die Dateien neu erstellen, falls eines der Abhängigkeiten (auswertung.py, L.txt, lambda.txt, I.txt) geändert wird.
+Schreibe ein `Makefile`, das beim Aufruf von `make` die Dateien `plot.pdf` und `parameter.txt` erstellt.
+Die Dateien sollten neu erstellt werden, falls eine der vorhandenen Abhängigkeiten (auswertung.py, L.txt, lambda.txt, I.txt) geändert wird.
+
+Fügen sie ein `clean` target hinzu, welches bei Aufruf von `make clean` den Ordner in den ursprünglichen Zustand vor dem Aufruf von `make` versetzt.
```
Comment (itoijala) [inline exercises-toolbox/5-make/aufgabe.txt]: "Fügen sie" → "Füge"
Oder "Sie" wenigstens groß.

Sollte es `clean`-Target sein?

File: latex/content/width.tex [hunk 3]
```diff
@@ -52,6 +52,7 @@ \subsection{Breites}
       \end{itemize}
     \end{column}
   \end{columns}
+  Ist hier natürlich etwas merkwürdig, da Beamer Slides bereits im Landscape Modus sind ...
```
Comment (itoijala) [inline latex/content/width.tex]: Beamer-Slides, Landscape-Modus

File: make/content/make.tex [hunk 4]
```diff
@@ -1,45 +1,55 @@
...
+    \begin{itemize}
+      \item Python-Skripe
+      \item \TeX{}
+      \item etc ...
+    \end{itemize}
   \end{itemize}
 \end{frame}
 
 \begin{frame}{Motivation}
+
```
Comment (itoijala) [inline make/content/make.tex]: Ist die leere Zeile nötig?

File: make/content/make.tex [hunk 5]
```diff
@@ -72,7 +92,7 @@
       plot.pdf: plot.py data.txt
           python plot.py
 
-      report.pdf: report.tex
+      report.pdf: report.tex
```
Comment (itoijala) [inline make/content/make.tex]: \s$ (Whitespace am Zeilenende), weg damit!

File: exercises-toolbox/5-make/aufgabe.txt (hunk 2 above)
Comment (mennthor) [inline exercises-toolbox/5-make/aufgabe.txt]: Ja, hast Recht. Werde das morgen noch ändern :+1:

File: latex/content/typography.tex [hunk 6]
```diff
@@ -49,30 +49,33 @@ \subsection{Ein bisschen Typografie}
 \end{frame}
 
 \begin{frame}[fragile]{Geschützte Leerzeichen}
+  Es gibt Leerzeichen an denen nicht umgebrochen werden soll. \\
```
Comment (itoijala) [inline latex/content/typography.tex]: Leerzeichen,

File: make/content/make.tex (hunk 5 above)
Comment (mennthor) [inline make/content/make.tex]: fixed

File: make/content/make.tex (hunk 4 above)
Comment (mennthor) [inline make/content/make.tex]: remoced line: fixed

File: latex/content/typography.tex (hunk 6 above)
Comment (mennthor) [inline latex/content/typography.tex]: fixed

File: latex/content/width.tex (hunk 3 above)
Comment (mennthor) [inline latex/content/width.tex]: fixed

File: exercises-toolbox/5-make/aufgabe.txt (hunk 2 above)
Comment (mennthor) [inline exercises-toolbox/5-make/aufgabe.txt]: fixed

File: latex/content/debug.tex (hunk 1 above)
Comment (mennthor) [inline latex/content/debug.tex]: Put color change in beamerstyle as suggested by mnoethe.
Set color to vertex light grey, to indicate typewriter with good contrast.

**Comment by MaxNoe** [comment]
Merge Merge.

Thanks a lot!
